api/tests/unit/projects/test_unit_projects_models.py

```diff
@@ -55,6 +56,35 @@ def test_get_segments_from_cache_set_not_called(project, segments, monkeypatch):
     mock_project_segments_cache.set.assert_not_called()
 
 
+@pytest.mark.django_db()
```

Comment:
We don't need this since the `project` and `segment` fixtures will require the `db` fixture already.